.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest a new feature for Weave Flux
+title: ''
+labels: [blocked-needs-validation, enhancement]
+assignees: ''
+
+---
+
+**Describe the feature**
+A clear and concise description of the feature.
+
+**What would the new user story look like?**
+How would the new interaction with Flux look like?
+0. What are the prerequisites for this?
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
```

Comment:
Click and scroll are not things that you would do with Flux, only with Weave Cloud :))